frontend/src/app/friends/page.js

```diff
@@ -39,29 +48,127 @@ export default function FriendsPage() {
     };
 
     const fetchFriends = async () => {
+        await getPendingFriends()
+        await getAcceptedFriends()
+    }
+
+    const getPendingFriends = async () => {
+        await friendService.getPendingFriends(USER_ID)
+            .then(res => {
+                console.log("fetching pending friends:", res.allPendingFriends)
+                setPendingFriends(res.allPendingFriends);
+            })
+            .catch(error => console.error('Error fetching pending friends:', error))
+    }
+
+    const getAcceptedFriends = async () => {
+        await friendService.getAcceptedFriends(USER_ID)
+            .then(res => {
+                console.log("fetching accepted friends:", res.allAcceptedFriends)
+                setAcceptedFriends(res.allAcceptedFriends);
+            })
+            .catch(error => console.error('Error fetching accepted friends:', error))
+    }
```

Comment:
Will have to change the response to use the array instead of allPendingFriends and allAcceptedFriends